guide/english/computer-hardware/gpu/index.md

```diff
@@ -1,84 +1,80 @@
 title: GPU
 ---
+## GPU
 
-# GPU
-GPU stands for Graphics Processing Unit. The majority of computers use these to render videos or play video games.
-
-## Table of Contents
-* [GPU](#gpu)
-* [Origin of GPU](#origin-of-gpu)
-* [GPU vs CPU](#gpu-vs-cpu)
-* [Evolution of GPU Architecture](#evolution-of-gpu-architecture)
-* [Basic Unified GPU Architecture Components](#basic-unified-gpu-architecture-components)
-
-![block diagram of gpu](https://upload.wikimedia.org/wikipedia/commons/thumb/c/cd/Generic_block_diagram_of_a_GPU.svg/650px-Generic_block_diagram_of_a_GPU.svg.png "By ScotXW - Own work, CC0, https://commons.wikimedia.org/w/index.php?curid=61055349")
-
-GPU stands for Graphics Processing Unit. Although they are not necessary for a computer to function, many computers have a dedicated graphics card for better performence rendering videos or playing video games.
+GPU stands for Graphics Processing Unit. Although they are not necessary for a computer to function, many computers have a dedicated graphics card for better performance rendering videos or playing video games.
 
 A GPU is like a CPU but has different strengths and weaknesses. CPUs are very good at running a couple of tasks very quickly. GPUs are much better at running many tasks at the same time but slower. A typical GPU can have more than 10,000 tasks running but to run so many tasks at the same time they must share memory and other resources. GPUs usually run very repetitive tasks over and over to save the CPU from wasting time. Some CPUs have built-in GPUs but having a separate GPU is almost always more powerful.
 
 GPUs can be used for computation as well as video rendering. Common ways to do this include OpenACC, CUDA, OpenCL, and OpenGL. Some applications include GPU implementations to reduce the amount of time the application takes to run.
 
-The GPU was originally used mainly for 3D game rendering to improve your resolution and framerate. Now these capabilities are being harnessed more broadly to improve computational workloads in many areas; for example financial modeling, cutting-edge scientific research and oil and gas exploration. GPU's are also used as a resource for bitcoin mining, since they are able to run repetitive tasks easily without straining the resources of the CPU, which allows you run an Operating System on the computer with a low end CPU while still being able to bitcoin mine using the GPU
+The GPU was originally used mainly for 3D game rendering to improve your resolution and framerate. Now, these capabilities are being harnessed more broadly to improve computational workloads in many areas; for example financial modeling, cutting-edge scientific research and oil and gas exploration. GPU's are also used as a resource for bitcoin mining since they are able to run repetitive tasks easily without straining the resources of the CPU, which allows you run an Operating System on the computer with a low-end CPU while still being able to bitcoin mine using the GPU
```

Comment:
Needs a period at the end of the last sentence after "GPU".

Reply:
@RandellDawson thanks for pointing out